### DISCUSSION TITLE
For DeathMessages to include custom mob name

### DISCUSSION THREAD

**Discussion by Eldilor99** [Zrips/CMI#7320] [Feature Requests]
Right now [MobName] displays only type of mob that killed a player(eg. Husk, Warden, Zombie, etc.), and it cannot be changed to [sourceDisplayName] (it doesn't work for mobs). It would be much appreciated if mob death messages could include their custom names as we have a lot of custom ones.

**Comment by Dagrond** [comment]
I would love this feature!
